src/Symfony/Component/Notifier/Bridge/Sendberry/SendberryTransport.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\Sendberry;
+
+use Symfony\Component\Notifier\Exception\IncompleteDsnException;
+use Symfony\Component\Notifier\Exception\TransportException;
+use Symfony\Component\Notifier\Exception\UnsupportedMessageTypeException;
+use Symfony\Component\Notifier\Message\MessageInterface;
+use Symfony\Component\Notifier\Message\SentMessage;
+use Symfony\Component\Notifier\Message\SmsMessage;
+use Symfony\Component\Notifier\Transport\AbstractTransport;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
+use Symfony\Contracts\HttpClient\Exception\TransportExceptionInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+/**
+ * @author Vasilij Duško <vasilij@prado.lt>
+ */
+final class SendberryTransport extends AbstractTransport
+{
+    protected const HOST = 'api.sendberry.com';
+
+    private string $username;
+    private string $password;
+    private string $authKey;
+    private string $from;
+
+    public function __construct(string $username, string $password, string $authKey, string $from, HttpClientInterface $client = null, EventDispatcherInterface $dispatcher = null)
+    {
+        $this->username = $username;
+        $this->password = $password;
+        $this->authKey = $authKey;
+        $this->from = $from;
+
+        parent::__construct($client, $dispatcher);
+    }
+
+    public function __toString(): string
+    {
+        return sprintf('sendberry://%s:%s@%s?auth_key=%s&from=%s', $this->username, $this->password, $this->getEndpoint(), $this->authKey, $this->from);
+    }
+
+    public function supports(MessageInterface $message): bool
+    {
+        return $message instanceof SmsMessage;
+    }
+
+    protected function doSend(MessageInterface $message): SentMessage
+    {
+        if (!$message instanceof SmsMessage) {
+            throw new UnsupportedMessageTypeException(__CLASS__, SmsMessage::class, $message);
+        }
+
+        if (!preg_match('/^[+]+[1-9][0-9]{9,14}$/', $this->from)) {
```

Comment:
don't you want to test against `$message->getPhone()` here?

Reply:
@welcoMattic Lithuanian numbers work. You can say your own and I will try to send you an SMS :)

Reply:
@StaffNowa I mean you are testing `$this->from` few lines below (https://github.com/symfony/symfony/pull/45195/files#diff-37eca20906cd07d4be220ae407f2e3f052f381bd13c0a29a576fedbfa1df2379R68), with another regex. Isn't it a mistake?